docs/reference/datatiers.asciidoc

```diff
@@ -2,40 +2,75 @@
 infrequently and not normally updated. To save space, you can keep
 <<fully-mounted,fully mounted indices>> of
 <<ilm-searchable-snapshot,{search-snaps}>> on the cold tier. These fully mounted
 indices eliminate the need for replicas, reducing required disk space by
 approximately 50% compared to the regular indices.
-* <<frozen-tier, Frozen tier>> nodes hold time series data that is accessed 
+* <<frozen-tier,Frozen tier>> nodes hold time series data that is accessed 
 rarely and never updated. The frozen tier stores <<partially-mounted,partially
 mounted indices>> of <<ilm-searchable-snapshot,{search-snaps}>> exclusively.
 This extends the storage capacity even further — by up to 20 times compared to
 the warm tier. 
 
-IMPORTANT: {es} generally expects nodes within a data tier to share the same 
-hardware profile. Variations not following this recommendation should be 
-carefully architected to avoid <<hotspotting,hot spotting>>.
+The way data tiers are used often depends on the data's category:
+
+- Content data remains on the <<content-tier,content tier>> for its entire
+data lifecycle. 
+
+- Time series data should progress through the 
+descending temperature data tiers (hot, warm, cold, and frozen) according to your 
+performance, resiliency, and data retention requirements. 
++ 
+You can automate these lifecycle transitions using the <<data-streams,data stream lifecycle>>, or custom <<index-lifecycle-management,{ilm}>>. 
 
-When you index documents directly to a specific index, they remain on content tier nodes indefinitely.
+[TIP]
+====
+A data tier's performance depends on its backing hardware profile. 
+For example hardware profiles, refer to Elastic Cloud's {cloud}/ec-reference-hardware.html[instance configurations].
 
-When you index documents to a data stream, they initially reside on hot tier nodes.
-You can configure <<index-lifecycle-management, {ilm}>> ({ilm-init}) policies
-to automatically transition your time series data through the hot, warm, and cold tiers
-according to your performance, resiliency and data retention requirements.
+Elastic generally assumes that lower temperature data tiers have an increased ratio of data storage to CPU or heap resources. This allows later tiers to gain more space for data storage at the cost of slower response times.
+
+Using the principle that lower-temperature tiers should hold less frequently updated and accessed data, your requests should be distributed to data tiers in the following approximate proportions. These proportions keep your clusters stable and highly responsive.
+	
+	- Search: 85% hot, 10% warm, 5% cold, and 1% frozen
+	- Ingest: 95% hot, 4% warm, 1% cold, and 0% frozen
+	
+	You can check how your access requests are distributed among your data tiers using the <<cat-thread-pool,CAT thread pools>> API.  If your lower temperature tiers are being accessed at higher proportions, then your cluster performance might be impacted. 
```

Comment:
I think looking through the cat threadpool API is a big request for an end user. It would be fairly easy to misunderstand, and since it's non-persistent it may give a very skewed view of a workload.

Reply:
That's fair! I'm curious what alternative investigation you'd recommend since it's a current user need? 

(I again may be ignorant of better ways. For the limited view I have: IME there's only hodge-podge answers like this outlined API currently but that would be a design improvement takeaway but not stop us from telling users the best they can introspect right now. A possible alternative would might be enabling Monitoring and then comparing node ingest rates; would that be better?)

Reply:
> A possible alternative would might be enabling Monitoring and then comparing node ingest rates; would that be better?

Doing it through monitoring would be better (not just ingest rates but also node load). We can think of threadpools as more of an "implementation detail" of the load/throughput rather than something we want them to rely upon.